### ISSUE TITLE
Undo is character-by-character rather than by change block

### ISSUE THREAD

**Issue by maackle** [deadfoxygrandpa/Elm.tmLanguage#112]
For all other syntax highlighters, I have normal undo, but for Elm, Undo happens one character at a time.

![undo](https://cloud.githubusercontent.com/assets/896230/19335883/9b8efa6a-90bb-11e6-860d-0c5069887fd8.gif)

I haven't seen anybody else mention this, maybe it's just me?

**Comment by branjwong** [comment]
Yeah, I am noticing it too on my machine. It's fixed when the the plugin is uninstalled and the problem comes back when I reinstall it.
